notifiers/elasticsearch/elasticsearch.go

```diff
@@ -23,32 +47,35 @@ type Settings struct {
 	CreateIndexTemplate bool              `field:"create_index_template" default:"true"`
 }
 
-const docType string = "/_doc"
-const indexTemplate string = "/_index_template/falco-talon"
+var parameters *Parameters
 
-var settings *Settings
+type Notifier struct{}
+
+func Register() *Notifier {
+	return new(Notifier)
+}
 
-func Init(fields map[string]interface{}) error {
-	settings = new(Settings)
-	settings = utils.SetFields(settings, fields).(*Settings)
-	if err := checkSettings(settings); err != nil {
+func (n Notifier) Init(fields map[string]interface{}) error {
+	parameters = new(Parameters)
+	parameters = utils.SetFields(parameters, fields).(*Parameters)
```

Comment:
see my other comment, this part as developed before the use of the validator, once we figured out how to deal with the default values, we'll be able to replace these blocks. just let's wait after the release to avoid to break the freeze with these invisible changes for the users